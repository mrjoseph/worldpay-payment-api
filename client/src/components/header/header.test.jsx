import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header';

describe('Header', () => {
  it('Should render the header without props', () => {
    const rendered = renderer.create(<Header />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
