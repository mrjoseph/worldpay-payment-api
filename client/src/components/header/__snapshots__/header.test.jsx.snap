// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render the header without props 1`] = `
<header
  className="row display-tr"
>
  <div
    className="display-td"
  >
    <img
      alt=""
      className="logo"
      src="worldpay.png"
    />
  </div>
</header>
`;
